src/YesSql.Core/Sql/SqlBuilder.cs

```diff
@@ -48,12 +48,12 @@ public SqlBuilder(string tablePrefix, ISqlDialect dialect)
         public void Table(string table, string alias = null)
         {
             FromSegments.Clear();
-            FromSegments.Add(_dialect.QuoteForTableName(_tablePrefix + table));
+            FromSegments.Add(_dialect.QuoteForColumnName(_tablePrefix + table));
```

Comment:
This one looks wrong

Reply:
Here the issue is that it passes there when we create a table alias.
So it does something like : 

`SELECT * FROM [dbo].[Table] AS [dbo].[Table1] INNER JOIN [dbo].[JoinedTable] AS [dbo].[JoinedTable1] ON [dbo].[JoinedTable1].[Column] = [dbo].[Table1].[Column]`

This is invalid SQL to use the schema prefix on table aliases.